### ISSUE TITLE
TAB key Focus on disabled elements - Pagination

### ISSUE THREAD

**Issue by iamvenkat45** [angular-ui/bootstrap#5151]
When use TAB key to focus elements, it is able to focus on disabled elements like “Previous” insteading of focus on only enabled elements like 1, 2, "Next".

Can we know on how to remove the focus on "previous" when this is disabled.

**Comment by icfantv** [comment]
@iamvenkat45, the pagination component does not have any key or focus handling code - everything you're talking about is handled via the browser.  For UI Bootstrap's part we do set the disabled class on the component when `ng-disabled` is `true`.

Some quick googling yielded [this link](http://stackoverflow.com/questions/3015523/remove-or-disable-focus-border-of-browser-via-javascript), which may be what you're looking for.

In the future, if you think you've found a bug, please follow [these instructions](https://github.com/angular-ui/bootstrap/blob/master/CONTRIBUTING.md#you-think-youve-found-a-bug) as we do not use our issues forum for support requests.  Rather, those should be directed towards SO as documented [here](https://github.com/angular-ui/bootstrap#need-help).  Thanks.

**Comment by iamvenkat45** [comment]
@icfantv thanks for the quick reply.

Disabled class is set on the component  but not on anchor tag and so the previous button is disabled but sill focussed.
The problem is when we use the TAB key, still the "previous" button is highlighted or focused on TAB navigation which shouldn't be.

As per your suggestion for <a href="http://stackoverflow.com/questions/3015523/remove-or-disable-focus-border-of-browser-via-javascript">this link</a>, This only makes the outline to none.
Actually the navigation should jump to next enabled link/element not with only making it focus none when disabled.

**Comment by icfantv** [comment]
@iamvenkat45, marking the anchor tag as disabled does not rectify the issue as you can [see here](http://plnkr.co/edit/lg4l1s2r11b48mNfpTKa).

I'm not yet convinced there's anything for us to do here.  @wesleycho or @Foxandxss, thoughts?

**Comment by wesleycho** [comment]
This does sound like an accessibility issue to me.

**Comment by wesleycho** [comment]
Forgot to put the issue # in the commit, but this is addressed via 1a870a3.
